packages/cli/src/index.ts

```diff
@@ -2,7 +2,18 @@
 
 import { Command } from 'commander';
 import { initCommand, devCommand, DevCommandOptions } from './commands';
+import { sync } from './commands/sync';
 
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** ‏use uuid from built-in `node:crypto` library instead